utils.py

```diff
@@ -11,9 +11,9 @@ def get_driver():
     return driver
 
 
-def get_soup_by_selenium_driver(website):
+def get_soup_by_selenium_driver(user_url):
```

Comment:
keep this function and variable naming generic here since it can be used anywhere. `website` or `url` better name